openwisp_notifications/models.py

```diff
@@ -31,6 +34,16 @@ def invalidate_cache(cls, user):
         """ invalidate cache for user """
         cache.delete(cls.COUNT_CACHE_KEY.format(user.pk))
 
+    @cached_property
+    def short_description(self):
+        return self.description.partition('\n')[0]
```

Comment:
I think the description can be loaded from the type configuration, like we do with chart configurations, see: 
https://github.com/openwisp/openwisp-monitoring/blob/master/openwisp_monitoring/monitoring/models.py#L298-L358

This way we can avoid repeating the same identical content in the DB.

Reply:
I tried to follow Laws of Demeter to maintain loose coupling. I have tried to use `NOTIFICATION_TYPES` only in `handlers.py` This was the only reason to implement it like that.